files-to-sync/scripts/test-unit.ts

```diff
@@ -0,0 +1,18 @@
+// This file is managed by macpro-mdct-core so if you'd like to change it let's do it there
+// node scripts/test-unit.ts
+import { execSync } from "node:child_process";
+import { chdir, cwd } from "node:process";
+
+let yarnCommand = "yarn install";
+if (process.env.CI === "true") {
+  yarnCommand = "yarn install --frozen-lockfile";
+}
```

Comment:
I like consts.  I don't know how everyone else feels.  Feel free to use this or don't use it.
`const yarn Command = process.env.CI === "true" ? "yarn install --frozen-lockfile" : "yarn install";`